lib/fs.js

```diff
@@ -101,6 +101,10 @@ function nullCheck(path, callback) {
   return true;
 }
 
+function isFd(path) {
+  return (path >>> 0) === path && path >= 0;
```

Comment:
Will all values where `typeof === 'number'` be treated as a number, all other values will be coerced to a string?